packages/labextension/pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = ["hatchling>=1.5.0", "jupyterlab>=4.0.0,<5", "hatch-nodejs-version"]
+build-backend = "hatchling.build"
+
+[project]
+name = "jupyterlab_jupytext"
```

Comment:
Personal nitpick, all packages and references are normalized to `jupyerlab-jupytext` (with `-` instead of `_`). I prefer to have the `project` name normalized like that as well for consistency.

Reply:
So do I, thanks